README.md

```diff
@@ -231,6 +231,15 @@ local t = function(str)
   return vim.api.nvim_replace_termcodes(str, true, true, true)
 end
 
+local check_back_space()
```

Comment:
That function definition doesn't seem right, should it not be
```lua
local check_back_space = function()
```

Reply:
yep. ... I delete _g miss funtion